<launch>

  <!-- set /global_example parameter -->  
  <group ns="search_port">

    <param name="open_climb_flag" value="0" />

    <param name="open_hold_flag" value="0"/>

    <param name="open_rotation_flag" value="0"/>





    <!--set /climb_pkg_para/close all 3dof driver-->
    <param name="close_all_3dof_climb_driver_flag" value="0"/>

    <!--set /climb_pkg_para/enable_control_stand_bar 1 enable 2 disable-->
    <param name="enable_control_stand_bar" value="0" />
    <!--set /climb_pkg_para/enable_control_rotation 1 enable 2 disable-->
    <param name="enable_control_rotation" value="0"/>
    <!--set /climb_pkg_para/enable_climb_control-->
    <param name="enable_climb_control" value="0"/>
    <!-- close driver second -->
    <param name="enable_second_control_stand_bar" value="0" />

    <param name="enable_second_climb_control" value="0"/>

    <param name="enable_third_stand_bar" value="0"/>
    <!-- get  climb_line_homing_abs_encode_data-->
    <param name="climb_line_homing_abs_encode_data" value="0.40"/>
    <!-- get  rotation_homing_abs_encode_data-->
    <param name="rotation_homing_abs_encode_data" value="44418"/>
    <!--set /climb_pkg_para/enable_control_stand_bar-->
    <param name="velocity_control_stand_bar" value="1000" />
    <!--set /climb_pkg_para/enable_control_rotation-->
    <param name="velocity_control_rotation" value="2000"/>
    <!--set /climb_pkg_para/enable_climb_control-->
    <param name="velocity_climb_control" value="1000"/>

    <!--set /climb_pkg_para/enable_control_stand_bar velocity-0.05/8=0.00625m/s 0.0-0.3-max neg-down pos-up-->
    <param name="distance_control_stand_bar" value="0" />
    <!--set /climb_pkg_para/enable_control_rotation -3.14-0.8 max pos-anticlockwise neg-clokwise-->
    <param name="rad_control_rotation" value="0"/>
    <!--set /climb_pkg_para/enable_climb_control 0-2.5m max pos-up neg-down-->
    <param name="distance_climb_control" value="0"/>    

    <node pkg="painting_robot_demo" type="climb_modbus_node.py" name="climb_modbus_node" output="screen" >
    </node>
    
  </group>
  
</launch>

